tests/foreman/cli/test_katello_agent.py

```diff
@@ -18,24 +18,23 @@
 """
 import pytest
 
+from pytest_fixtures.core.contenthosts import register_host_custom_repo
```

Comment:
**RedFlag!** Not recommended to import the fixtures. Please use them directly in fixture.

Reply:
@jyejare This is a function that is written there in pytest_fixtures, and not a fixture, anyway I also think this should be moved to Satellite class in robottelo.hosts or maybe some helper for content hosts tests.
I think this is no RedFlag for this PR, and this can be handled separately in PR?

Reply:
Not sure what could be the consequences/side-effects for importing something from fixture modules which are being initiated from the central conftest.py.

But if you want to move this to ContentHost class where u might get some similar methods written there where you can merge into this.

OR Simply turn that function into function level fixture :)

Reply:
@jyejare Updated, moved this func to robotello hosts class